crates/rpc/src/handlers/pool.rs

```diff
@@ -55,3 +55,54 @@ pub async fn post_voluntary_exits(
 
     Ok(HttpResponse::Ok())
 }
+
+/// GET /eth/v1/beacon/pool/proposer_slashings
+#[get("/beacon/pool/proposer_slashings")]
+pub async fn get_pool_proposer_slashings(
+    slashing_pool: Data<Arc<SlashingPool>>,
+) -> Result<impl Responder, ApiError> {
+    let slashings = slashing_pool.get_all_proposer_slashings();
+    Ok(HttpResponse::Ok().json(DataResponse::new(slashings)))
```

Comment:
```suggestion
#[get("/beacon/pool/proposer_slashings")]
pub async fn get_pool_proposer_slashings(
    slashing_pool: Data<Arc<SlashingPool>>,
) -> Result<impl Responder, ApiError> {
    Ok(HttpResponse::Ok().json(DataResponse::new(slashing_pool.get_all_proposer_slashings())))
```

we might as well inline this